chef_master/source/upgrade_server.rst

```diff
@@ -189,6 +195,8 @@ To upgrade to Chef server 12 from a high availability Chef server server, do the
 
       $ chef-server-ctl reconfigure
 
+#. On each server, download the desired Chef server version from the `Chef Server Downloads <https://downloads.chef.io/chef-server>`__ page.
```

Comment:
Could we say to copy the previously downloaded package? Saves us some bandwidth costs of repeat downloads.